docs/developer/continuous-deployment/continuous_deployment.md

```diff
@@ -229,13 +229,11 @@ Configure the following GitHub secrets which are required by the CD pipeline:
 | `ACR_NAME`                    | The name of the Azure Container Registry to deploy. Use only lowercase letters and numbers. |
 | `TERRAFORM_STATE_STORAGE_ACCOUNT` | The name of the storage account used to store the Terraform state container, e.g. *mvdterraformstates*. |
 | `TERRAFORM_STATE_CONTAINER` | The name of the container used to store the Terraform state blob, e.g. *mvdterraformstates*. |
-
-Update the value of the `CD_RESOURCES_PREFIX` env in the [cloud-cd.yaml](../../../.github/workflows/cloud-cd.yaml) file.
-This prefix allows to have unique resource names across fork repositories when running CD workflows.
+| `CD_RESOURCES_PREFIX_REVERSED` | A string, that when reversed, is a globally unique sequence of 3 to 4 lowercase letters and digits, starting with a letter, that will be used to name Azure resources. For example, the value `vmde` will result in the string `edmv` being used in Azure resource names. This prefix allows to have unique resource names across fork repositories when running CD workflows.<br />*Note: The reason the prefix is stored as a reversed string is to avoid the value being excluded from GitHub job outputs.* |
```

Comment:
We need to do something for automated initial setup scripts as well.

e.g. This is the script where it adds secret in github https://github.com/agera-edc/MinimumViableDataspace/blob/main/resources/setup_azure_ad/set-gh-secrets.sh

Reply:
Why? The user should follow the instructions in the README file. The script is an additional tool, but not meant to replace the README

Reply:
Because as per section https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/blob/main/docs/developer/continuous-deployment/continuous_deployment.md#update-github-secrets it says that

_This can be done manually using the Github website (see documentation). Alternatively there is a shell script that will set all Github Secrets automatically. From the resources/setup_azure_ad directory simply execute ./set-gh-secrets.sh on a shell._

Which to me gives an impression that if I execute ./set-gh-secrets.sh then I will have all the required github secrets.

So, my suggestion is either we update the README to highlight that CD_RESOURCES_PREFIX_REVERSED will not be created by shell script and needs to done manually or this script takes care of this on as well.

Reply:
Good catch, I've updated the section